reference/concepts/imperative_programming.md

```diff
@@ -0,0 +1,7 @@
+## Imperative programming
```

Comment:
There is an existing document describing declarative programming: https://github.com/exercism/v3/blob/master/reference/paradigms/imperative.md Could you remove this document from the PR?